tutorials/hist/candlehisto.C

```diff
@@ -49,7 +49,7 @@ void candlehisto()
    char myopt[6][16] = {"1000000","2000000","3000000","1112111","112111","112111"};
    for (int i = 0; i < 6; i++) {
       c3->cd(i+1);
-      char str[16];
+      char str[17];
       sprintf(str, "candlex(%s)",myopt[i]);
```

Comment:
@pcanal The strings are used with TH2I which only accepts traditional strings. Would you recommend transforming the TString to a char array for each argument that requires a `const char *` instead of a `TString`?